src/test/rustdoc-gui/search-result-go-to-first.goml

```diff
@@ -0,0 +1,12 @@
+// This test ensures that the "go_to_first" feature is working as expected.
+
+// First, we check that the first page doesn't have the string we're looking for to ensure
+// that the feature is changing page as expected.
+goto: file://|DOC_PATH|/test_docs/index.html
```

Comment:
should we also go to a search page without go_to_first?

Reply:
There is `search-result-description.goml` which does that already. :)

Reply:
Pop in a comment saying that and r=me

Reply:
Ok!

Reply:
Actually the test will be more complete if we add that as an intermediate check.